ci-operator/step-registry/hypershift/install/hypershift-install-commands.sh

```diff
@@ -13,6 +13,10 @@ if [ "${ENABLE_HYPERSHIFT_OPERATOR_DEFAULTING_WEBHOOK}" = "true" ]; then
   EXTRA_ARGS="${EXTRA_ARGS} --enable-defaulting-webhook=true"
 fi
 
+if [ "${ENABLE_HYPERSHIFT_OPERATOR_VALIDATING_WEBHOOK}" = "true" ]; then
+  EXTRA_ARGS="${EXTRA_ARGS} --enable-validating-webhook=true"
```

Comment:
I think for boolean flags, `--enable-validating-webhook` is enough to set it. no need for `=true`

Reply:
Thanks @orenc1 - but I always prefer to be explicit.